OutOfSchool/OutOfSchool.RazorTemplatesData/Views/Emails/ApplicationStatusUpdateEmail/ApplicationStatusUpdateEmail.cshtml

```diff
@@ -0,0 +1,57 @@
+﻿@using OutOfSchool.RazorTemplatesData.Models.Emails
+@using OutOfSchool.RazorTemplatesData.Models.Shared
+@using OutOfSchool.Services.Enums
+@model ApplicationStatusViewModel
+
+@{
+    ViewData["EmailTitle"] = "";
+    var ending = "";
+
+    if (Model.ChildGender == Gender.Male) 
+    {
+        ending = "ий";
+    }
+    else if (Model.ChildGender == Gender.Female)
+    {
+        ending = "а";
+    }
+    else if (Model.ChildGender == null)
+    {
+        ending = "ий/a";
+    }
```

Comment:
(Code maintainability/style) This code is duplicated. 
As its only one dependency is `Model.ChildGender` - you could introduce it as autocalculated property in the model, and put all logic there. Ofc there are more ways to do the same. Please choose any and avoid duplication.

Reply:
Thanks